### DISCUSSION TITLE
Can't open file

### DISCUSSION THREAD

**Discussion by jekv2** [bromite/bromite#1163] [Q&A] [answered]
I don't know how to use the submit bug. 

Here.

91.0.4472.50
Android 11 sm-g975u

When download location is set to SD aka external storage, download an apk, try to open it from bromite download windows "can't open file".

**Comment by WebworkrNet** [comment]
Same problem under Android 10 (Samsung Galaxy XCover 4S)

**Comment by uazo** [comment, marked as answer]
@csagan5 I think this may be the mistake

https://github.com/bromite/bromite/commit/9a1f7c424299c66fc1edca1d68260e072c940832#diff-dae8e7dc793bd897909816bc2a1793199f7021686927d858cd9a49e38f269244

in v87 for `Disable-smart-selection-by-default.patch` we changed

```
 const base::Feature kChromeSmartSelection{"ChromeSmartSelection",
-                                          base::FEATURE_ENABLED_BY_DEFAULT};
+                                          base::FEATURE_DISABLED_BY_DEFAULT};
```
to
```
 const base::Feature kDownloadFileProvider{"DownloadFileProvider",
-                                          base::FEATURE_ENABLED_BY_DEFAULT};
+                                          base::FEATURE_DISABLED_BY_DEFAULT};
```

**Reply by csagan5** [reply]
Indeed; I will change both lines so that this does not happen again.

This must be an artifact of fuzzy patch applying.

The feature was removed in https://github.com/chromium/chromium/commit/da1d809c003749846cb4ade8c11b6d038e44416b; I will restore it.

**Comment by jekv2** [comment]
Fails to download 
<a href='https://postimg.cc/SYnQ1bdH' target='_blank'><img src='https://i.postimg.cc/SYnQ1bdH/Screenshot-20210527-114655-Bromite.png' border='0' alt='Screenshot-20210527-114655-Bromite'/></a>
